README.md

```diff
@@ -62,7 +62,7 @@ Let's see what's in the `wine` package:
 >>> wine.quality.red._keys()
```

Comment:
You'll need to update the lines above - at least `wine`, but maybe remove the calls to `._keys()` now that `repr` shows them.